### PR TITLE
(plugin) check kubernetes cluster through azure API

### PR DESCRIPTION
**Pull request by lchrdn**
Plugin to check Kubernetes cluster through Azure API

### CODE REVIEW

File: cloud/azure/compute/aks/plugin.pm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+use base qw(centreon::plugins::script_custom);
+
+sub new {
+    my ( $class, %options ) = @_;
+    my $self = $class->SUPER::new( package => __PACKAGE__, %options );
+    bless $self, $class;
+
+    $self->{version} = '0.1';
+    %{ $self->{modes} } = (        
+	'allocatableresources'  => 'cloud::azure::compute::aks::mode::allocatableresources',
```
Comment (garnier-quentin) [inline cloud/azure/compute/aks/plugin.pm:34]: indent error

**Comment by Sims24** [comment]
Awesome! Thanks.

File: cloud/azure/compute/aks/plugin.pm (hunk 1 above)
Comment (garnier-quentin) [reply cloud/azure/compute/aks/plugin.pm:34]: maybe we could add a dash for the mode: 'allocatable-resources'

**Review by Sims24 — APPROVED** [review]

File: cloud/azure/compute/aks/plugin.pm [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        'cpu'                   => 'cloud::azure::compute::aks::mode::cpu',
+        'discovery'             => 'cloud::azure::compute::aks::mode::discovery',
+        'storage'               => 'cloud::azure::compute::aks::mode::storage',
+        'traffic'               => 'cloud::azure::compute::aks::mode::traffic',
+        'unneedednodes'         => 'cloud::azure::compute::aks::mode::unneedednodes'
```
Comment (garnier-quentin) [inline cloud/azure/compute/aks/plugin.pm:39]: maybe we could add a dash: unneeded-nodes

**Pull request merged**